src/core/lib/surface/completion_queue.cc

```diff
@@ -186,16 +179,83 @@ void non_polling_poller_shutdown(grpc_pollset* pollset, grpc_closure* closure) {
   }
 }
 
+bool maybe_event_engine_poller_can_get_pollset() {
+  return !grpc_event_engine::experimental::UsePollsetAlternative();
+}
+
+bool non_polling_poller_can_never_get_pollset() { return false; }
```

Comment:
This should probably move up above maybe_event_engine_poller_can_get_pollset().

Reply:
Moved, thanks!